src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/EditorTests.java

```diff
@@ -183,4 +185,27 @@ public void MercuryCuratedEditorTest_003_addAndSaveItemToOptionalSection() {
         result
     );
   }
+
+  @Test(groups = "MercuryCuratedEditorTest_004")
+  @Execute(onWikia = MercuryWikis.MERCURY_EMPTY_CC_EDITOR,
+      onDriver = Driver.ChromeMobileMercury)
+  public void MercuryCuratedEditorTest_004_saveNewPhoto() {
+    CuratedMainPagePageObject curatedMainPagePageObject = new CuratedMainPagePageObject(driver);
+    EditorHomePageObject editorHomePageObject = new EditorHomePageObject(driver);
+
+    Boolean result = !curatedMainPagePageObject.isCuratedContentVisible();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "result" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)